code/Data exploration.R

```diff
@@ -37,6 +38,8 @@ lapply(package_list, library, character.only = T)
 temperature_nsw    <- fread("data/temperature_nsw.csv")
 totaldemand_nsw    <- fread("data/totaldemand_nsw.csv")
 forecastdemand_nsw <- fread("data/forecastdemand_nsw.csv")
```

Comment:
its annoying how this isn't reflected in the repo - I'll see if theres a way around this. otherwise we'll have to manually add this file ourselves